tests/integration/test_permissions.py

```diff
@@ -57,7 +58,7 @@ def _create_and_index_record(record):
     return record
 
 
-@pytest.fixture(scope='function')
+@pytest.fixture(scope='session')
```

Comment:
I got a PIDAlreadyExists error because every time the `sample_record` was trying to create the same record again i think.